include/llvm-c/BLAKE3.h

```diff
@@ -0,0 +1,79 @@
+/*===-- llvm-c/blake3.h - BLAKE3 C Interface ----------------------*- C -*-===*\
```

Comment:
Seems like this file should be renamed to `llvm-c/blake3.h`.. Could you double check why the name is capitalized in the commit?

It seems like in `lib/llvm/Support/BLAKE3/blake3_impl.h` and `include/llvm/Support/BLAKE3.h` we have the lowercased version `#include "llvm-c/blake3.h"`